migrations/1_initialize_schema.up.sql

```diff
@@ -481,6 +481,53 @@ CREATE TABLE issue_ticket (
   CONSTRAINT issue_ticket_pk PRIMARY KEY (issue_id, ticket_id)
 );
 
+----------------------------------------------------------------------------------------
+
+
+------------------------------ ACL Security --------------------------------------------
+
+create table acl_sid(
+    id bigserial not null primary key,
+    principal boolean not null,
+    sid varchar(100) not null,
+    constraint unique_uk_1 unique(sid,principal)
+);
+
+create table acl_class(
+    id bigserial not null primary key,
+    class varchar(100) not null,
+    constraint unique_uk_2 unique(class)
+);
+
+create table acl_object_identity(
+    id bigserial primary key,
+    object_id_class bigint not null,
+    object_id_identity varchar(36) not null,
+    parent_object bigint,
+    owner_sid bigint,
+    entries_inheriting boolean not null,
+    constraint unique_uk_3 unique(object_id_class,object_id_identity),
```

Comment:
Minor. It looks better when fk (pk, un) name consists keys name.

Reply:
The schema is provided by Spring 
https://docs.spring.io/spring-security/site/docs/4.0.x/reference/html/appendix-schema.html